### DISCUSSION TITLE
Search suggestions

### DISCUSSION THREAD

**Discussion by maximka76667** [gradejs/gradejs#99] [Ideas]
Hey! I really like your new design and I appreciate that you took my suggestion into consideration!👍

I tried the new version and it works fine too, but then I realized that I don't know what to look for.
You have a list of suggestions below the search form, but I find it rather modest and limited (5 on PC and 3 on mobile).

I'm not sure how difficult this will be for your project, or if you're excited about adding new UI components after the redesign, but I think it would be great to have a larger list of suggestions and show them while user is searching (e.g. Google search)

![suggestion](https://user-images.githubusercontent.com/74974283/194328986-c486aba3-12d0-4647-90fd-efbf56616238.jpg)

I won't be upset if right now you are more focused on the technical part than on the UI, but I would be glad if you could add that too.
Hope I don't bore you with my suggestions :)

**Comment by spalt08** [comment]
Thanks for the feedback! We have a mockup for that, and going to deploy it in a few weeks
<img width="858" alt="image" src="https://user-images.githubusercontent.com/5869473/197059203-5de3f630-1de8-4278-81fa-7ee0ec5e1b5f.png">
